src/main/resources/languages/messages_en.yml

```diff
@@ -42,6 +43,12 @@ guide:
 
   cheat:
     no-multiblocks: '&4You cannot cheat in Multiblocks, you have to build them!'
+
+  backpack:
+    invalid-id: '&4The id must be a non-negative integer!'
```

Comment:
Rather use the word "number", "integer" does not appear to be so user-friendly. Also maybe call it "backpack id" rather than just "id"